test/src/waittask.spec.ts

```diff
@@ -730,9 +661,7 @@ describe('waittask specs', function () {
         return (error = error_);
       });
       expect(error).toBeInstanceOf(puppeteer.errors.TimeoutError);
-      expect(error?.message).toContain(
-        'waiting for selector `.//div` failed: timeout 10ms exceeded'
```

Comment:
Are the waiting failure details still in the error message?

Reply:
See above.